ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -129,6 +129,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8
+- as: e2e-aws-ipi-ovn-hypershift-p2
```

Comment:
BTW, we already have `e2e-aws-ipi-hypershift-p2` in 4.12.
Since OVN is the default in 4.12, `e2e-aws-ipi-ovn-hypershift-p2` and `e2e-aws-ipi-hypershift-p2` are duplicate.

Reply:
`e2e-aws-aws-ipi-hypershift-p2` no longer exists, we changed `e2e-aws-ipi-ovn-hypershift-mgmt-p2` for better understanding

Reply:
We use the same workflow `cucushift-installer-rehearse-aws-ipi-ovn-hypershift` to set up two clusters, and use cluster1 to test test-suite1, and use cluster2 to test test-suite2. This waste of resources and time (duplicate installation).
Is this the direction we are going ? @jhou1 @chengzhang1016

Reply:
@kuiwang02 I see both you have the comments to merge into one tests based on this workflow. I prefer to keep them all for different test scenarios.
1) The previously `e2e-aws-ipi-ovn-hypershift-mgmt-p2` is use to test hypershift hosted cluster including control plane and worker nodes.  This testsuit  includes hypershift core functions testcases together with OCP e2e on hosted clusters.  It is valuable to let us know if there is any error/bug in hypershift side for self-managed and even managed hypershift service.

2) The new added test in this PR is based on the same workflow. But it is for the work moving jenkins CI to prow. That is to replace [128_IPI on AWS & OVN & HyperShift](https://gitlab.cee.redhat.com/aosqe/ci-profiles/-/blob/master/4.12/128_IPI%20on%20AWS%20%26%20OVN%20%26%20HyperShift.install.yaml).
This CI is used to test management cluster instead of hypershift hosted clusters. It is a general test for OCP component e2e just to make sure the management cluster should not be affected by hypershift. Because hypershift control plane pods & operator pods are just applications in the mgmt cluster.
This scenario may include many testcases.  

In short, the first one is for hypershift hosted cluster CI. It is used to check HyperShift core functions & the hosted cluster functions are as expected
Another is for OCP CI with hypershift installed. It is used to check OCP itself before a new version of OCP is released.

Reply:
@heliubj18 I understand your reply is similar with https://github.com/openshift/release/pull/33023#issuecomment-1275494495, and my reply is https://github.com/openshift/release/pull/33023#issuecomment-1275497716